Controller/BaseController.php

```diff
@@ -2,6 +2,9 @@
 
 namespace Razorpay\Magento\Controller;
 
+// require in case of zip installation without composer
+require_once __DIR__ . "/../../Razorpay/Razorpay.php";
```

Comment:
@ankitdas13 , please remove this one. This only needs to be added in case of zip.

Reply:
ok i will remove this